StRoot/StETofHitMaker/StETofHitMaker.h

```diff
@@ -139,12 +154,21 @@ class StETofHitMaker: public StMaker {
 
     std::map< UInt_t, Double_t > mSigVel; // signal velocities in each detector
 
-    Double_t mSoftwareDeadTime;      // dead time introduced in software to reject after pulses on the same channel
-    Bool_t   mDoClockJumpShift;      // correct for clock jumps on one side
+    Double_t mSoftwareDeadTime;            // dead time introduced in software to reject after pulses on the same channel
+    Bool_t   mDoClockJumpShift;            // correct for clock jumps on one side
+    Bool_t   mDoDoubleClockJumpShift;      // correct for clock jumps on both sides
 
     std::map< Int_t, Int_t >     mClockJumpDirection;  // stores direction of clock jump for time correction
 
+    std::map< Int_t, int >     mModMatrix;  // stores mode of modification for hits on striplevel (flip)
+
+    double mGet4doublejumpTmin;                                            // cutoff for double jump correction
+    std::map< Int_t, bool  >                    mGet4doublejumpFlag;      // get4  jumpflag
+    std::map< Int_t, std::vector < float > >    mGet4doublejumpTimes;     // get4  time of last n hits
```

Comment:
In what units this time is measured? Do you really need a vector of real values?